### ISSUE TITLE
**Using conda-smithy to manage your CI** section is there in USER Docs. 

### ISSUE THREAD

**Issue by PrernaSingh587** [conda-forge/conda-forge.github.io#1405]
- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:
https://conda-forge.org/docs/user/ci-skeleton.html#using-conda-smithy-to-manage-your-ci
**Using conda-smithy to manage your CI** section is there in USER Docs. I am a little confused as to why would a new comer or a user would be concerned with CI management of the package. Maybe I am not totally aware of the workflow, so I'm opening this issue to get the opinions of all.

**Comment by beckermr** [comment]
This section is not about using the conda package but about something totally different. Let's leave it where it is.

**Comment by jakirkham** [comment]
We just had a new user run into this webpage and use it to try and re-render: https://github.com/conda-forge/conda-smithy/issues/1842#issuecomment-2101647788

Can completely understand where they are coming from as the "User Documentation" has a lot of info for helping new users. For example "A brief Introduction", "Becoming Involved", "How to get help", "FAQ", etc.. So it might be reasonable to think "Using conda-smithy to manage your CI" being in "User Documentation" is intended for new conda-forge users

However "Using conda-smithy to manage your CI" is talking about how to configure feedstock external to conda-forge, which is pretty advanced material. It is kind of jarring to have this next to intro material. Tbh am surprised this hasn't caused more problems for new users

Was about to file a new issue for this, when I found this old one. Think we should reconsider this page's location

cc @jaimergp (in case this is something you have thoughts on as well)

**Comment by matthewfeickert** [comment]
> Can completely understand where they are coming from as the "User Documentation" has a lot of info for helping new users. For example "A brief Introduction", "Becoming Involved", "How to get help", "FAQ", etc.. So it might be reasonable to think "Using conda-smithy to manage your CI" being in "User Documentation" is intended for new conda-forge users

I agree, but my view is slightly different than that. I'm a maintainer of multiple conda-forge recipes for more than 5 years, but I generally only interact with conda-smithy when I'm reviewing a new release PR (and so I am new to viewing it as anything outside of arguments to the right incantations to talk with a bot (`conda-forge-admin`)). The "problem" in https://github.com/conda-forge/conda-smithy/issues/1842#issuecomment-2101651063 was that when asked to certify that

> Solution to issue cannot be found in the documentation.

there is no "conda-smithy docs". There is the (very nice! and terribly huge!) conda-forge docs

![image](https://github.com/conda-forge/conda-forge.github.io/assets/5142394/f31f4949-3aff-47cd-8c9c-36d7b41ee8f2)

The distinction between "user" and "maintainer" here is a bit weird too, as while I understand(/guess ?) that the "user"/"maintainer" divide is between package maintainers using conda-forge for distribution and the people maintaining conda-forge, by and large, anytime I am looking in the docs on how to get a non-trivial (aka, something beyond `grayskull pypi` generated) recipe to build, I'm always looking in either the maintainer section of the docs or reading the [`conda-build` metadata docs](https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#defining-metadata-meta-yaml).

So given that last statement, it would also probably help to have on both the [user docs](https://conda-forge.org/docs/user/) and [maintainer docs](https://conda-forge.org/docs/maintainer/) top level pages as "Who is this for" summary, to help steer people a bit.

**Comment by jaimergp** [comment]
Thanks for the excellent feedback. Hopefully we can tackle this as part of https://github.com/conda-forge/conda-forge.github.io/issues/2164. I agree with your points.

conda-smithy has some docs in [its README](https://github.com/conda-forge/conda-smithy), but not sure if that's enough.

**Comment by matthewfeickert** [comment]
> Hopefully we can tackle this as part of https://github.com/conda-forge/conda-forge.github.io/issues/2164. I agree with your points.

Thanks and that's great to see the work being done on the Issue. :+1: 

> conda-smithy has some docs in [its README](https://github.com/conda-forge/conda-smithy), but not sure if that's enough.

Here (Issue https://github.com/conda-forge/conda-smithy/issues/1842) I think the problem was that I only have `micromamba` and `pixi` installed, and so no base conda environment, which I believe is required for the `conda smithy rerender` command to work. I don't think that caveat shows up in the README.
